agent/src/main/java/com/microsoft/applicationinsights/agent/internal/logger/InternalAgentLogger.java

```diff
@@ -85,7 +85,7 @@ public synchronized void initialize(String loggerLevel) {
             } else {
                 loggingLevel = LoggingLevel.valueOf(loggerLevel.toUpperCase());
             }
-        } catch (Throwable t) {
+        } catch (Exception e) {
```

Comment:
@littleaj these were the places you referred that doesn't need throwable correct?